src/bindings/recompose.ts

```diff
@@ -218,3 +232,90 @@ function withPropFromContext(
     }
   }
 }
+
```

Comment:
I'm not sure if the whole withState implementations should stay here or if I should rather move it to separate file. @adam-stanek?

Reply:
You can leave it as it is for now. I will probably restructuralize it a bit in the future :)